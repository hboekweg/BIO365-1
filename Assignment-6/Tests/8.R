source("8.R")