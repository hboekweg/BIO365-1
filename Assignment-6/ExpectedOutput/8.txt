# A tibble: 1 x 9
  otu    count kingdom  phylum    class      order    family    genus   species
  <chr>  <int> <chr>    <chr>     <chr>      <chr>    <chr>     <chr>   <chr>  
1 taxa21 15003 Bacteria Proteoba… Alphaprot… Sphingo… Sphingom… Sphing… <NA>   
